source/core/Array.cpp

```diff
@@ -189,11 +189,11 @@ VIREO_FUNCTION_SIGNATUREV(ArrayFillNDV, ArrayFillNDVParamBlock)
             tempDimensionLengths[i] = 0;
     }
 
-    _Param(ArrayOut)->ResizeDimensions(numDimensionInputs, tempDimensionLengths, false);
-
-    IntIndex totalLength = _Param(ArrayOut)->Length();
-    TypeRef eltType = array->ElementType();
-    eltType->MultiCopyData(_ParamPointer(InitialValue), array->RawBegin(), totalLength);
+    if (_Param(ArrayOut)->ResizeDimensions(numDimensionInputs, tempDimensionLengths, false)) {
+        IntIndex totalLength = _Param(ArrayOut)->Length();
```

Comment:
During resizing due to some failure if there is no action done we have to skip succeeding dependent code. There was a test failure due to this.

Reply:
The vireo test mentioned the above comment failing after emsdk update.
I think the intention of this test to check the overflow error report generated [here](https://github.com/ni/VireoSDK/blob/c8eb11d197b09f7ed2100bdb01cd48cd8f2fa003/source/core/TypeAndDataManager.cpp#L162).
so for the particular case, `ResizeDimensions` won't happen because of the expected memory allocation error. We are accessing that array later in this function which leads to segmentation faults so added check before using it.

Reply:
Devika, Milan and I discussed about this also. We should revert this change because the node should fail with an error or the execution should stop on a fatal failure such as memory allocation failure. It is incorrect for the execution to continue. Devika will make the changes and try to fix the JS test to handle the exception.

Reply:
Reverted these changes and created [tech debt](https://ni.visualstudio.com/DevCentral/_workitems/edit/1507808/) item to track changes to stop the execution after fatal failure such as memory allocation failure.